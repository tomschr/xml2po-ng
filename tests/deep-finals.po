# Ок
msgid ""
msgstr ""
"Project-Id-Version: проба\n"
"POT-Creation-Date: 2005-03-27 13:12+0200\n"
"PO-Revision-Date: 2005-03-27 13:12+0200\n"
"Last-Translator: Данило <EMAIL@ADDRESS>\n"
"Language-Team: српски <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: deep-finals.xml:2 (para)
msgid "beautiful"
msgstr "дивно"

#: deep-finals.xml:2 (para)
msgid "Something <something>nice and <placeholder-1/></something> ha!"
msgstr "Нешто <something>лепо и <placeholder-1/></something> ха!"

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: deep-finals.xml:0 (None)
msgid "translator-credits"
msgstr "Данило"
