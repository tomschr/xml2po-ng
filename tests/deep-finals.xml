<example>
<para>Something <something>nice and <para>beautiful</para></something> ha!</para>
</example>
