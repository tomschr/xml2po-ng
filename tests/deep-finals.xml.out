<?xml version="1.0" encoding="utf-8"?>
<example>
<para>Нешто <something>лепо и <para>дивно</para></something> ха!</para>
</example>
